devices/pine64-pinephone/firmware/default.nix

```diff
@@ -7,12 +7,13 @@
 runCommandNoCC "pine64-pinephone-firmware" {
   src = fetchgit {
     url = "https://megous.com/git/linux-firmware";
-    rev = "4ec2645b007ba4c3f2962e38b50c06f274abbf7c";
-    sha256 = "0mx5h2r7j5bik4wdkgdyzjpj1x6fx2y4p8y1ir4ic76902xhipr6";
+    rev = "6e8e591e17e207644dfe747e51026967bb1edab5";
+    hash = "sha256-TaGwT0XvbxrfqEzUAdg18Yxr32oS+RffN+yzSXebtac=";
```

Comment:
Just checked

 - https://nixos.org/manual/nix/stable/release-notes/rl-2.2.html
 - https://github.com/NixOS/nixpkgs/blob/master/lib/minver.nix

So this is fine.